docs/macros.py

```diff
@@ -70,7 +70,7 @@ def format_funcs(function_methods: List[str]) -> str:
         # Requires Python linebreak and markdown linebreak (two spaces)!
         # "\n\t\t• `abc`  \n\t\t• `cde`  \n\t\t• `cde`
 
-        formatted = [f"\n\t\t&emsp; • `.{func}()`  " for func in function_methods]
+        formatted = [f"\n\t\t&emsp; • `.{func}()`  " for func in function_methods]  # noqa: E702
```

Comment:
to suppress flake8's warning:
` E702 multiple statements on one line (semicolon)`

Reply:
Again checked on fresh installation of environment, worked without complaints

Reply:
@javidq this is strange.
Can you confirm your environment is using python 3.12?

Reply:
My environment uses 3.9 as the minimal required

Reply:
Makes sense.
Everything works smoothly for python 3.9, 3.10, and 3.11

But for 3.12, all these changes are needed.

Reply:
The issue was fixed in flake 6.1.0, let's update to the latest version